driver-core/src/main/com/mongodb/bulk/IndexRequest.java

```diff
@@ -512,4 +513,30 @@ public IndexRequest collation(final Collation collation) {
         this.collation = collation;
         return this;
     }
+
+    /**
+     * Gets the wildcard projection of a wildcard index
+     *
+     * @return the wildcard projection
+     * @mongodb.server.release 4.2
+     * @mongodb.driver.manual TODO
```

Comment:
Waiting for permalink from docs team.

Reply:
Yeah, I agree. I didn't plan to commit this until I hear back from the DOCS team.  If I don't get anything soon I'll remove this and commit.

Reply:
cool - LGTM once updated then.

Reply:
I got tired of waiting so just removed.